lib/galaxy/files/templates/examples/production_onedata.yml

```diff
@@ -0,0 +1,30 @@
+- id: onedata
+  version: 0
+  name: Onedata Storage
+  description: |
+    This template allows connecting to Onedata servers.
```

Comment:
chyba warto rozszerzyć o to że to dla files source i że do wgrywania danych z onedata (można wspomnieć że readonly)

Reply:
EDIT dodałem wymaganie wersji